frontend/src/resources/resource.ts

```diff
@@ -77,7 +77,7 @@ export function getResourceApiPath(options: {
         path = join(path, options.kind.toLowerCase() + 's')
     }
 
-    return path
+    return path.replace(/\\/g, '/')
```

Comment:
When would we have a case with paths that have "\".  Seems like that code should be fixed and not patched here.

Reply:
I'll back it out--I only need it on Windows